java-api-examples/Makefile

```diff
@@ -0,0 +1,56 @@
+
+ENTRY_POINT = ./
+
+LIB_FILES = \

```

Comment:
I suggest the following to make the code shorter
```makefile
DIR := ../sherpa-onnx/java-api/src/com/k2fsa/sherpaonnx
LIB_FILES := $(DIR)/EndpointRule.java
LIB_FILEs += $(DIR)/FeatureConfig.java
...
```

Also, please sort filenames alphabetically. That is,
EndpointRule.java is placed before FeatureConfig.java.
```